Mapping/AttributeMetadata.php

```diff
@@ -236,6 +245,22 @@ public function getIri()
     }
 
     /**
+     * @return boolean
+     */
+    public function isIdentifier()
+    {
+        return $this->isIdentifier;
```

Comment:
According to Symfony CS, names must be:
$this->identifier
isIdentifier()
setIdentifier()